kinode/src/kernel/process.rs

```diff
@@ -347,8 +347,11 @@ impl ProcessState {
             let this_blob = blob.clone();
             let self_sender = self.self_sender.clone();
             let original_target = t::Address::de_wit(target.clone());
+            let send_to_terminal = self.send_to_terminal.clone();
+            let our = self.metadata.our.clone();
             let timeout_handle = tokio::spawn(async move {
                 tokio::time::sleep(std::time::Duration::from_secs(timeout_secs)).await;
+                print(&send_to_terminal, 0, format!("{our}: timed out")).await;
```

Comment:
yeah, controversial indeed.. i don't think this is a good timeout-tracking solution

Reply:
is this actually necessary or just a visual debugging tool?

Reply:
It is quite hard to debug timeouts without an indication of what is going on. We could increase verbosity level or try to figure out some alternative way to do help devs debug, but the status quo of not informing devs is bad (I spent a day tracking down timeout-related bug because it wasn't clear to me what timeouts were happening, whereas if there were some type of print, it would've been much faster).

Reply:
It's hard for me to relate since timeouts *do* show up in the event loop trace, and unless there's a kernel bug in how they get sent back to processes, they should come up in the receive() message queue. In almost all the programs I've written, I have a print that shows any errors that come into my process. Is this an issue that's specific to runtime extension development?

Reply:
You are correct. I think maybe this was due to the specific path I followed in bringing `run-tests` into a more usable state or perhaps getting confused about multiple timeouts hitting. Regardless, I've removed the print in f2b8d38